### DISCUSSION TITLE
Recommended way to install latest dev version?

### DISCUSSION THREAD

**Discussion by davidwebca** [tailwindlabs/tailwindcss#7712] [General]
Hi,

I don't know if it's a problem with the current version and dependency changes, but when I try to pull tailwindcss with yarn through git (yarn add tailwindcss@https://github.com/tailwindlabs/tailwindcss#master --dev), it resolves the package, but doesn't contain the lib folder. My brain is foggy and I can't figure out what's the build process to release a new npm version, so maybe that I could fix it temporarily to install the latest, but is there any documentation out there on what is the recommended process if we want to test the dev version from git?

The reason is that it happens quite often that there's a particular issue that I'm watching, is getting solved and then I'd like to test before it gets released.

Thanks a lot and sorry if it's evident! 😅

**Comment by adamwathan** [comment]
Hey! We actually automate an `insiders` release every time a commit is merged to master, so you can always get what's in master by installing Tailwind this way:

```
npm install -D tailwindcss@insiders
```

That should get you what you're looking for 💪

**Reply by davidwebca** [reply]
Great, thanks so much! Exactly what I was looking for. 👍
